Dockerfile

```diff
@@ -22,10 +24,15 @@ RUN pip install --upgrade pip \
 RUN curl -L https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash; \
     helm init --client-only
 
-# Install kubectl
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl; \
-    mv kubectl /usr/bin/kubectl; \
-    chmod +x /usr/bin/kubectl
+# Install cfssl and cfssljson
+RUN curl -sSL https://pkg.cfssl.org/R1.2/cfssl_linux-amd64 > /usr/bin/cfssl \
+ && curl -sSL https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64 > /usr/bin/cfssljson \
+ && chmod +x /usr/bin/cfssl /usr/bin/cfssljson
+
+# Install kubectl and kubeadm
+RUN curl -L https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl > /usr/bin/kubectl \
```

Comment:
you can use `curl -L --remote-name <url>` here, so you can save the stdout redirection